src/ecc/tb/test_vectors/ecc_drbg_mbedtls.hex

```diff
@@ -7,6 +7,8 @@ BB9C3A2F061E8D7014278DD51E66A918A6B6F9F1C1937312D4E7A921B18EF0F41FDD401D9E771850
 93799D551226362834F60F7B945290B7CCE6E99601FB7EBD026C2E3C445D3CD9B65068DAC0A848BE9F0560AA758FDA27
 E548E535A1CC600E133B5591AEBAAD78054006D752D0E1DF94FBFA95D78F0B3F8E81B9119C2BE008BF6D6F4E4185F87D
 000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
+8A80623F88D24575C412C2F53975893EC14F6EC2D6E29CA0D11E6A43969D34F2175F72C8DE5F94B734406A2F84E1D15B
```

Comment:
I kept privkeyB random since privkeyA/pubkeyA are also random. Only hash, seed, nonce, IV seem to be fixed. Do we want to have privkeyB also fixed to 00s in test vector #1 and FFs in #2?